ansible/roles/ml-analytics-scripts/tasks/main.yaml

```diff
@@ -0,0 +1,62 @@
+---
+
+- name: Ensure Scripts Directory Exists
+  file:
+    path: "{{ BASEPATH }}"
+    state: directory
+    owner: "{{ USER }}"
+    group: "{{ USER }}"
+
+- name: Git checkout
+  git:
+    repo: 'https://github.com/shikshalokam/sl-sunbird-implementation-utils.git'
+    dest: "{{ BASEPATH }}"
+    version: "{{ ml_scripts_version }}"
+    depth: "1"
+    force: yes
+
+- name: Ensure config Directory Exists
+  file:
+    path: "{{ BASEPATH }}/common_config"
+    state: directory
+    owner: "{{ USER }}"
+    group: "{{ USER }}"
+
+- name: Save config file
+  template: src=config.j2 dest="{{ BASEPATH }}/common_config/config.ini" mode=0644
+
+- name: CHANGE THE OWNERSHIP FOR THIS {{ BASEPATH }} DIRECTORY
+  file:
+    path: "{{ item }}"
+    mode: "0755"
+    recurse: yes
+    state: directory
+    owner: "{{ USER }}"
+    group: "{{ USER }}"
+  loop:
+    - "{{ BASEPATH }}"
+
+- name: Delete the virtualenv DIR
```

Comment:
As we are removing the python venv as part of provision, it will have impact if the same is provisioned in the existing DP spark server. Should this need to be provisioned in a separate spark server allocated for ML workloads?